build.gradle

```diff
@@ -102,7 +102,7 @@ ext {
 }
 
 group = "mesosphere"
-version = "0.7.10-SNAPSHOT"
+version = "0.7.11-SNAPSHOT"
```

Comment:
Please either keep it at 0.7.10-SNAPSHOT, or do a release of 0.7.10, then bump to 0.7.11-SNAPSHOT

https://docs.google.com/document/d/1S-zNC80CeW3ETHJGpgt22ALiEQtz4otLKYqVWj1nA4I/edit